generator/csv/fields.csv

```diff
@@ -1,1114 +1,1114 @@
-SaveSystem,battle_start_fadeout,f,UInt8,0x71,1,
-SaveSystem,battle_start_fadein,f,UInt8,0x72,1,
-SaveSystem,battle_end_fadeout,f,UInt8,0x73,1,
-SaveSystem,battle_end_fadein,f,UInt8,0x74,1,
-SaveSystem,teleport_allowed,f,Boolean,0x79,True,
-SaveSystem,escape_allowed,f,Boolean,0x7A,True,
-SaveSystem,save_allowed,f,Boolean,0x7B,True,
-SaveSystem,menu_allowed,f,Boolean,0x7C,True,
-SaveSystem,background,f,String,0x7D,,string
-SaveSystem,save_count,f,Int32,0x83,0,
-SaveSystem,save_slot,f,Int32,0x84,1,
-SaveSystem,atb_mode,f,Enum<SaveSystem_AtbMode>,0x8C,0,ATB mode of RPG 2003 battle system.
-SaveScreen,tint_finish_red,f,Int32,0x01,100,int
-SaveScreen,tint_finish_green,f,Int32,0x02,100,int
-SaveScreen,tint_finish_blue,f,Int32,0x03,100,int
-SaveScreen,tint_finish_sat,f,Int32,0x04,100,int
-SaveScreen,tint_current_red,f,Double,0x0B,100.0,double
-SaveScreen,tint_current_green,f,Double,0x0C,100.0,double
-SaveScreen,tint_current_blue,f,Double,0x0D,100.0,double
-SaveScreen,tint_current_sat,f,Double,0x0E,100.0,double
-SaveScreen,tint_time_left,f,Int32,0x0F,0,int
-SaveScreen,flash_continuous,f,Boolean,0x14,False,int
-SaveScreen,flash_red,f,Int32,0x15,0,int
-SaveScreen,flash_green,f,Int32,0x16,0,int
-SaveScreen,flash_blue,f,Int32,0x17,0,int
-SaveScreen,flash_current_level,f,Double,0x18,0.0,double
-SaveScreen,flash_time_left,f,Int32,0x19,0,int
-SaveScreen,shake_continuous,f,Boolean,0x1E,False,int
-SaveScreen,shake_strength,f,Int32,0x1F,0,int
-SaveScreen,shake_speed,f,Int32,0x20,0,int
-SaveScreen,shake_position,f,Int32,0x21,0,int
-SaveScreen,shake_position_y,f,Int32,0x22,0,int - unused
-SaveScreen,shake_time_left,f,Int32,0x23,0,int
-SaveScreen,pan_x,f,Int32,0x29,0,int
-SaveScreen,pan_y,f,Int32,0x2A,0,int
-SaveScreen,battleanim_id,f,Int32,0x2B,0,int - battle animation ID
-SaveScreen,battleanim_target,f,Int32,0x2C,0,int - battle animation target
-SaveScreen,battleanim_frame,f,Int32,0x2D,0,int - frame count - Ref<Animation>? FIXME
-SaveScreen,unknown_2e_battleanim_active,f,Int32,0x2E,0,int - unsure FIXME
-SaveScreen,battleanim_global,f,Boolean,0x2F,False,int - battle animation global scope
-SaveScreen,weather,f,Int32,0x30,0,int
-SaveScreen,weather_strength,f,Int32,0x31,0,int
-SavePicture,name,f,String,0x01,'',string
-SavePicture,start_x,f,Double,0x02,0.0,double
-SavePicture,start_y,f,Double,0x03,0.0,double
-SavePicture,current_x,f,Double,0x04,0.0,double
-SavePicture,current_y,f,Double,0x05,0.0,double
-SavePicture,fixed_to_map,f,Boolean,0x06,False,bool
-SavePicture,current_magnify,f,Double,0x07,-1.0,double
-SavePicture,current_top_trans,f,Double,0x08,0.0,double
-SavePicture,transparency,f,Boolean,0x09,False,bool
-SavePicture,current_red,f,Double,0x0B,-1.0,double
-SavePicture,current_green,f,Double,0x0C,-1.0,double
-SavePicture,current_blue,f,Double,0x0D,-1.0,double
-SavePicture,current_sat,f,Double,0x0E,-1.0,double
-SavePicture,effect_mode,f,Int32,0x0F,0,int
-SavePicture,current_effect,f,Double,0x10,0.0,double
-SavePicture,current_bot_trans,f,Double,0x12,0.0,double
```

Comment:
Seems these have been doubles before at least.

Reply:
Excel clobbered the doubles, I was using it to make these changes.

There is also a code change in `generate.py` here to always write doubles as a double constant in the C++ code regardless of whether the csv has a decimal point or not.

Reply:
Yep, saw that change of course, that's why I made the comment... ;-)